.gitignore

```diff
@@ -1,4 +1,3 @@
-target
```

Comment:
Why remove target?

Reply:
JNAerator generates the classes that we need right there under target (see files in this PR).

That being said, I can exclude the other auto-generated stuff from JNAerator we don't need under `target`, namely `*.class`, `*.xml`, etc...).